src/config/config.ts

```diff
@@ -181,9 +172,8 @@ export class Config implements IConfig {
   }
 
   async loadDevPlugins() {
-    if (this.options.devPlugins !== false) {
-      // do not load oclif.devPlugins in production
-      if (hasManifest(path.join(this.root, 'oclif.manifest.json'))) return
+    // do not load oclif.devPlugins in production
+    if (this.options.devPlugins !== false && process.env.NODE_ENV === 'development') {
```

Comment:
Why would devPlugins be false?

Can you put the env check in an `isProd()` method?